solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -79,31 +79,29 @@ function install_yq() {
 function update_solution_config(){
     pushd $SCRIPT_LOCATION/k8_cortx_cloud
         echo > solution.yaml
-        yq e -i '.solution.namespace = "default"' solution.yaml
+        NAMESPACE=$NAMESPACE yq e -i '.solution.namespace = env(NAMESPACE)' solution.yaml
         deployment_method=$DEPLOYMENT_METHOD yq e -i '.solution.deployment_type = env(deployment_method)' solution.yaml
         
         yq e -i '.solution.secrets.name = "cortx-secret"' solution.yaml
-        yq e -i '.solution.secrets.content.kafka_admin_secret = "Seagate@123"' solution.yaml
-        yq e -i '.solution.secrets.content.consul_admin_secret = "Seagate@123"' solution.yaml
-        yq e -i '.solution.secrets.content.common_admin_secret = "Seagate@123"' solution.yaml
+        yq e -i '.solution.secrets.content.kafka_admin_secret = "null"' solution.yaml
+        yq e -i '.solution.secrets.content.consul_admin_secret = "null"' solution.yaml
+        yq e -i '.solution.secrets.content.common_admin_secret = "null"' solution.yaml
         yq e -i '.solution.secrets.content.s3_auth_admin_secret = "ldapadmin"' solution.yaml
```

Comment:
looks like this is also changed to null in v0.3.0

Reply:
We use `s3_auth_admin_secret` and `csm_mgmt_admin_secret` in automated IO testing hence not changing them to null as of now. 

Also as per Walter - 
`
Default passwords are no longer provided in solution.example.yaml (in solution.secrets.content).  Best practice is either to leave these all blank (and let deploy-cortx-cloud.sh generate passwords), or to specify passwords for the default IAM account (s3_auth_admin_secret) and CSM (csm_mgmt_admin_secret) and let deploy-cortx-cloud.sh generate the rest.
`